### DISCUSSION TITLE
Recommended way of using eksctl for IAC

### DISCUSSION THREAD

**Discussion by rverma-dev** [weaveworks/eksctl#5504] [Q&A]
Wondering what the recommended way of using eksctl in IAC/automation is. It seems like one tool that can't be automated using 
cloudformation/CDK/step-functions etc...

There is a terraform provider https://github.com/mumoshu/terraform-provider-eksctl but it looks mostly unmaintained as of now.
Please recommend a stable way to use eksctl to bootstrap 100s of clusters in an automated way.

**Comment by danielloader** [comment]
Keep in mind eksctl uses cloudformation for state management and resource deployment.
